src/components/TitledRoute/TitledRoute.js

```diff
@@ -12,7 +12,7 @@ import { withStripes } from '../../StripesContext';
 class TitledRoute extends React.Component {
   static propTypes = {
     component: PropTypes.element,
-    computedMatch: PropTypes.element,
+    computedMatch: PropTypes.object,
```

Comment:
Could you instead say `PropTypes.oneOfType([PropTypes.element, PropTypes.object])`?

Reply:
Sure! (I have no idea what this was trying to achieve, so I'm fine by it.)